packages/core/src/templates/contract/partials/event_types.handlebars

```diff
@@ -1,31 +1,41 @@
-export const enum {{contractName}}Events {
-  {{#each events}}
-  {{this.name}} = "{{this.name}}",
-  {{/each}}
-}
+// tslint:disable:no-namespace
+export namespace {{contractName}} {
+  export const enum Events {
```

Comment:
Instead of forcing existence of all `const enums`, removing `const` modifier from here might be less hacky

Reply:
its weird that there is a distinction between enum and const enum